TrippyGL.Fonts.Building/FontGlyphSource.cs

```diff
@@ -118,8 +115,13 @@ private IPathCollection[] CreatePaths(out Color?[][] colors, out System.Drawing.
             {
                 char c = (char)(i + FirstChar);
                 glyphRenderer.Reset();
-                GlyphInstance glyphInstance = FontInstance.GetGlyph(c);
-                glyphInstance.RenderTo(glyphRenderer, Size, new Vector2(0, glyphRenderY), new Vector2(DrawDpi, DrawDpi), 0);
+                GlyphMetrics glyphMetrics = fontMetrics.GetGlyphMetrics(new CodePoint(c), ColorFontSupport.None).FirstOrDefault();
```

Comment:
`Font` has `GetGlyphs` here which returns the collection of glyphs matching a given codepoint. You can use the returns `Glyph` instances to render to the renderer.

Reply:
I'll give it a shot!

Thanks a lot for your help and sorry for the delay answering; I'm travelling and I've also been sick for a few days. I want to give this a look but it might be a while before I get the time to do so

Reply:
Absolutely no problem at all!

I'm more than happy to wait for your feedback. It's important that we get this right together.